src/Interactive/HostTest/NuGetPackageResolverTests.cs

```diff
@@ -114,17 +166,24 @@ public void UnhandledException()
         public void ParsePackageNameAndVersion()
         {
             ParseInvalidPackageReference("A");
-            ParseInvalidPackageReference("A.B");
-            ParseInvalidPackageReference("A/");
-            ParseInvalidPackageReference("A//1.0");
-            ParseInvalidPackageReference("/1.0.0");
-            ParseInvalidPackageReference("A/B/2.0.0");
+            ParseInvalidPackageReference("A/1");
+            ParseInvalidPackageReference("nuget");
+            ParseInvalidPackageReference("nuget:");
+            ParseInvalidPackageReference("NUGET:");
+            ParseInvalidPackageReference("nugetA/1");
+            ParseInvalidPackageReference("nuget:A");
+            ParseInvalidPackageReference("nuget:A.B");
+            ParseInvalidPackageReference("nuget:A/");
+            ParseInvalidPackageReference("nuget:A//1.0");
+            ParseInvalidPackageReference("nuget:/1.0.0");
+            ParseInvalidPackageReference("nuget:A/B/2.0.0");
```

Comment:
Nevermind, I see the whitespace tests below.  We allow arbitrary whitespace then?